server/lib/tuist/storage.ex

```diff
@@ -225,4 +245,19 @@ defmodule Tuist.Storage do
 
     size
   end
+
+  defp s3_config_and_bucket(actor) do
+    if use_tigris?(actor) do
+      {ExAws.Config.new(:s3_tigris), Environment.s3_bucket_name(:tigris, Environment.decrypt_secrets())}
```

Comment:
```suggestion
      {ExAws.Config.new(:s3_tigris), Environment.s3_bucket_name(:tigris)}
```
Should be able to use the default \\